build/Dockerfile

```diff
@@ -23,3 +23,5 @@ COPY job-templates/* /default-job-templates/
 COPY _output/bin/eunomia ${OPERATOR}
 
 ENTRYPOINT ["/usr/local/bin/eunomia-operator"]
+
+USER 12345
```

Comment:
We also need to make sure that the group is not root. So how about this instead ...

```
RUN groupadd -r eunomia && useradd --no-log-init -r -g eunomia eunomia
USER eunomia
```

Reply:
Sure, done.